source/view-model.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by Ksenia Annenko on 24.02.2017.
+ */
+
+/*********************** View-Model **********************/
+"use strict";
+function ViewModel() {
+    this.init = function () {
+        let firstTestPlayer = 'First';
+        let secondTestPlayer = 'Second';
+        let game = new Game(false, firstTestPlayer, secondTestPlayer);
+        this.gameInfo = new GameInfo(game);
+        game.startTurn();
+        this.finishTurn = function (id) {
+            game.finishTurn(id);
+        };
+    };
+}
+
+function GameInfo(game){
+    this.getFieldCells = function(){
+        return game.getFieldCells();
+    };
+    this.getCurrentStatus = function () {
```

Comment:
I think we should follow the same code style in our code. Here we have two different function declarations: function () and function(). Please select one of them and use everywhere in your code.